pycognito/__init__.py

```diff
@@ -277,8 +277,15 @@ def verify_token(self, token, id_name, token_use):
         # Compute and verify at_hash (formerly done by python-jose)
         if "at_hash" in verified:
             alg_obj = jwt.get_algorithm_by_name(header["alg"])
-            digest = alg_obj.compute_hash_digest(self.access_token)
-            at_hash = base64.urlsafe_b64encode(digest[: (len(digest) // 2)]).rstrip("=")
+            try:
```

Comment:
This is supposed to deal with functions expecting strings vs bytes differently with updated versions of the cryptography dependency.

The try-catch block allows the digest to be computed in either case and then the at_hash step is split based on the type returned by the digest computation since this changes depending on so that the strip step does not fail.

The idea was not to change function but to make it robust to some typing mismatches that were occurring. That said, I am not experiencing the issue now with python 3.11.9 and the updated dependencies.

Reply:
Okey, that is why I asked, beacuse after looking at different versions, there is no differences between them in what they return or expect as inputs.

Reply:
Going to close this one for now, as it no longer seems to be needed.
Thanks anyway 👍